tests/server/test_prometheus_exporter.py

```diff
@@ -0,0 +1,50 @@
+import os
+from tempfile import TemporaryDirectory
+from unittest import mock
+
+import pytest
+
+from mlflow.server.prometheus_exporter import activate_prometheus_exporter
+
+tmpdir = TemporaryDirectory()
+
+
+@pytest.fixture()
+def app():
+    from mlflow.server import app
+
+    ctx = app.app_context()
+    ctx.push()
+    yield app
+    ctx.pop()
```

Comment:
```suggestion
    with app.app_context():
        yield app
```

Can we use `with` to simplify the code?